src/ui/modal/modalRemoveFile/view.jsx

```diff
@@ -32,20 +32,28 @@ function ModalRemoveFile(props: Props) {
       <Form onSubmit={() => deleteFile(uri, deleteChecked, claimIsMine ? abandonChecked : false)}>
         <FormField
           name="file_delete"
-          label={__('Also delete this file from my computer')}
+          label={__('Delete this file from my computer')}
           type="checkbox"
           checked={deleteChecked}
           onChange={() => setDeleteChecked(!deleteChecked)}
         />
 
         {claimIsMine && (
-          <FormField
-            name="claim_abandon"
-            label={__('Abandon the claim for this URI')}
-            type="checkbox"
-            checked={abandonChecked}
-            onChange={() => setAbandonChecked(!abandonChecked)}
-          />
+          <div>
+            <FormField
+              name="claim_abandon"
+              label={`${__('Abandon on blockchain')} (${__('reclaim')} ${claim.amount} ${__('LBC')})`}
```

Comment:
To work better with other language strings, this should be:
`__("Abandon on blockchain (reclaim %amount% LBC)", { amount: claim.amount })`

Reply:
That makes sense to me.  Updated!